modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingFlowComposable.kt

```diff
@@ -125,21 +129,7 @@ private fun Content(
                     if (state.isNewAccount) {
                         onAccountCreated()
                     } else {
-                        when (flow) {
-                            OnboardingFlow.InitialOnboarding,
-                            OnboardingFlow.LoggedOut,
-                            is OnboardingFlow.PlusAccountUpgrade,
-                            is OnboardingFlow.PlusUpsell -> exitOnboarding()
```

Comment:
Having these two flows _not_ exit onboarding after signing in with Google is the fix this PR is making.